hazelcast/src/main/java/com/hazelcast/map/impl/tx/TransactionalMapProxy.java

```diff
@@ -105,47 +105,47 @@ public boolean isEmpty() {
     }
 
     @Override
-    public Object get(Object key) {
+    public V get(Object key) {
         return get(key, false);
     }
 
-    public Object get(Object key, boolean skipNearCacheLookup) {
+    public V get(Object key, boolean skipNearCacheLookup) {
```

Comment:
let's keep this as Object return type, it serves for both client and member proxies and for client we sent serialized form.